lib/human_console_player.rb

```diff
@@ -7,6 +7,10 @@ def initialize(mark, ui)
     @ui = ui
   end
 
+  def ready?
```

Comment:
This ready? method returning true is duplicated in the HumanConsolePlayer and the computer_player. Could it be shared somehow?